docs/dev/style-guide.rst

```diff
@@ -0,0 +1,53 @@
+Documentation Style Guide
+=========================
+
+This document will serve as the canonical place to define how we write documentation at Read the Docs.
+The goal is to have a shared understanding of how things are done,
+and document the conventions that we follow.
+
+Let us know if you have any questions or something isn't clear.
+
+The brand
+---------
+
+We are called **Read the Docs**.
+The ``the`` is not capitalized.
+
+We do however use the acronym **RTD**.
+
+Titles
+------
+
+For page titles we use sentence case.
```

Comment:
Awesome - should we take an approach to use sentence case in all open and upcoming PRs and file an issue for "100% consistency in all titles" that we solve at the end of Iteration 1?

Reply:
Yes, we chose that we wanted this and until the decision was made to actually do it, we would keep consistency with the _old_ Title Case for titles.

I think this is the milestone where we use sentence cases moving forwards :+1: